src/undo.h

```diff
@@ -50,7 +66,24 @@ class CTxInUndo
         if (nHeight > 0)
             ::Unserialize(s, VARINT(this->nVersion), nType, nVersion);
         ::Unserialize(s, REF(CTxOutCompressor(REF(txout))), nType, nVersion);
+
+        if ((this->nVersion & 0x7f) == (SC_CERT_VERSION & 0x7f)) {
```

Comment:
make this more clear by checking also nHeight > 0 
(we store that info only if coin was pruned and this is the only case of having nHeight > 0)
unserialize also the backwardTrasferStartIdx